Languages/IronPython/IronPython.Modules/datetime.cs

```diff
@@ -1486,6 +1494,22 @@ public class time : ICodeFormattable {
 
             #endregion
 
+            public object __format__(CodeContext/*!*/ context, string dateFormat) {
+                if (string.IsNullOrEmpty(dateFormat)) {
+                    return PythonOps.ToString(context, this);
+                }
+                else {
+                    // If we're a subtype, there might be a strftime overload,
+                    // so call it if it exists.
+                    if (GetType() == typeof(time)) {
+                        return this.strftime(context, dateFormat);
```

Comment:
That was a copy/paste from date.__format__, but I've cleaned up both instances in the following commit.